### DISCUSSION TITLE
force error

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/deepmd-kit#592] [Q&A]
My force error is 0.159 ev/A, is it suitable or not?

<img width="757" alt="截屏2021-05-06 20 48 13" src="https://user-images.githubusercontent.com/16278257/117300712-678f8900-aeac-11eb-9df8-5feccd4d40f4.png">

**Comment by AnguseZhang** [comment]
It depends. Criterions are different on different systems. You should find some references and do relevant research.

**Comment by McAlexDudu** [comment]
Ok, Thanks👌🏻

Sent from Pr_Alex
